src/panels/lovelace/views/hui-view-background.ts

```diff
@@ -0,0 +1,130 @@
+import { css, LitElement, nothing } from "lit";
+import type { CSSResultGroup, PropertyValues } from "lit";
+import { customElement, property } from "lit/decorators";
+import type { HomeAssistant } from "../../../types";
+import type { LovelaceViewBackgroundConfig } from "../../../data/lovelace/config/view";
+
+@customElement("hui-view-background")
+export class HUIViewBackground extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) background?:
+    | string
+    | LovelaceViewBackgroundConfig
+    | undefined;
+
+  protected render() {
+    return nothing;
+  }
+
+  private _applyTheme() {
+    const computedStyles = getComputedStyle(this);
+    const themeBackground = computedStyles.getPropertyValue(
+      "--lovelace-background"
+    );
+
+    const fixedBackground = this._isFixedBackground(
+      this.background || themeBackground
+    );
+    const viewBackground = this._computeBackgroundProperty(this.background);
+    this.toggleAttribute("fixed-background", fixedBackground);
+    this.style.setProperty("--view-background", viewBackground);
+
+    const viewBackgroundOpacity = this._computeBackgroundOpacityProperty(
+      this.background
+    );
+    this.style.setProperty("--view-background-opacity", viewBackgroundOpacity);
+  }
+
+  private _isFixedBackground(
+    background?: string | LovelaceViewBackgroundConfig
+  ) {
+    if (typeof background === "string") {
+      return background.split(" ").includes("fixed");
+    }
+    if (typeof background === "object" && background.attachment === "fixed") {
+      return true;
+    }
+    return false;
+  }
+
+  private _computeBackgroundProperty(
+    background?: string | LovelaceViewBackgroundConfig
+  ) {
+    if (typeof background === "object" && background.image) {
```

Comment:
If background is an option, can we set the css individually instead of using the shorthand format?